packages/@react-native/tester/js/examples/Modal/ModalCustomizable.js

```diff
@@ -0,0 +1,343 @@
+    const colorStyle = {
+      color: this.state.active ? '#fff' : '#000',
+    };
+    return (
+      <TouchableHighlight
+        onHideUnderlay={this._onUnhighlight}
+        onPress={this.props.onPress}
+        onShowUnderlay={this._onHighlight}
+        style={[styles.button, this.props.style]}
+        underlayColor="#a9d9d4">
+        <Text style={[styles.buttonText, colorStyle]}>
+          {this.props.children}
+        </Text>
+      </TouchableHighlight>
+    );
+  }
+}
+
+const supportedOrientationsPickerValues = [
+  ['portrait'],
+  ['landscape'],
+  ['landscape-left'],
+  ['portrait', 'landscape-right'],
+  ['portrait', 'landscape'],
+  [],
+];
+
+class ModalExample extends React.Component<{...}, $FlowFixMe> {
+  state: $FlowFixMe = {
+    animationType: 'none',
+    modalVisible: false,
+    transparent: false,
+    hardwareAccelerated: false,
+    statusBarTranslucent: false,
+    presentationStyle: 'fullScreen',
+    selectedSupportedOrientation: '0',
+    currentOrientation: 'unknown',
+    action: '',
+  };
+
+  _setModalVisible = visible => {
+    this.setState({modalVisible: visible});
+  };
+
+  _setAnimationType = type => {
+    this.setState({animationType: type});
+  };
+
+  _toggleTransparent = () => {
+    this.setState({transparent: !this.state.transparent});
+  };
+
+  _toggleHardwareAccelerated = () => {
+    this.setState({hardwareAccelerated: !this.state.hardwareAccelerated});
+  };
+
+  _toggleStatusBarTranslucent = () => {
+    this.setState({statusBarTranslucent: !this.state.statusBarTranslucent});
+  };
+
+  renderSwitch(): React.Node {
+    if (Platform.isTV) {
+      return null;
+    }
+    if (Platform.OS === 'android') {
+      return (
+        <>
+          <Text style={styles.rowTitle}>Hardware Accelerated</Text>
+          <Switch
+            value={this.state.hardwareAccelerated}
+            onValueChange={this._toggleHardwareAccelerated}
+          />
+          <Text style={styles.rowTitle}>Status Bar Translucent</Text>
+          <Switch
+            value={this.state.statusBarTranslucent}
+            onValueChange={this._toggleStatusBarTranslucent}
+          />
+          <Text style={styles.rowTitle}>Transparent</Text>
+          <Switch
+            value={this.state.transparent}
+            onValueChange={this._toggleTransparent}
+          />
+        </>
+      );
+    }
+    return (
+      <Switch
+        value={this.state.transparent}
+        onValueChange={this._toggleTransparent}
+      />
+    );
+  }
+
+  render(): React.Node {
+    const modalBackgroundStyle = {
+      backgroundColor: this.state.transparent
+        ? 'rgba(0, 0, 0, 0.5)'
+        : '#f5fcff',
+    };
+    const innerContainerTransparentStyle = this.state.transparent
+      ? {backgroundColor: '#fff', padding: 20}
+      : null;
+    const activeButtonStyle = {
+      backgroundColor: '#ddd',
+    };
+
+    return (
+      <ScrollView contentContainerStyle={styles.ScrollView}>
+        <Modal
```

Comment:
Do we run this example in RnTester? If so, how does `Modal` behave (is that implemented in RNW?)

Reply:
Modal is unimplemented in RNW.